src/catalog/src/memory/objects.rs

```diff
@@ -649,10 +649,13 @@ pub struct Table {
 }
 
 impl Table {
-    // The Coordinator controls insertions for tables (including system tables),
-    // so they are realtime.
     pub fn timeline(&self) -> Timeline {
-        Timeline::EpochMilliseconds
```

Comment:
I'm honestly worried about how many places in the code blindly assumes that all tables are on the `EpochMillisecond` timeline. Nothing to do I suppose but see what breaks.